Sara/index.js

```diff
@@ -0,0 +1,10 @@
+// 1. Write program to picking random country from countries.
+let randomCountry = ['Germany', 'Italy', 'USA'] ;
```

Comment:
If you dont use array to solve this exercise the below code could be the solution! 
let countries = "Germany Spain Italy USA";
let value = countries.split(" ");
let pickCountry = value[Math.floor(Math.random() * 4)];
console.log("picking random country ==>", pickCountry);